src/components/learner-credit-management/members-tab/tests/MembersTab.test.jsx

```diff
@@ -741,7 +751,7 @@ describe('<BudgetDetailPage />', () => {
     const toggleAllRowsSelected = screen.getByTitle('Toggle All Current Page Rows Selected');
     userEvent.click(toggleAllRowsSelected);
 
-    const downloadButton = screen.getByText('Download (1)');
+    const downloadButton = screen.getByText('Download 1');
```

Comment:
based on the feedback above, this should be updated to `Download (1)` to match figma mocks.